house_interiors_2.logic

```diff
@@ -0,0 +1,18 @@
+area HouseInteriors2:
+  room Anju
+  room Dampe
+  room DrLeft
+  room Gorman1A
+  room Gorman1B
+  room Gorman1C
+  room Gorman2A
+  room Gorman2B
+  room Gorman2C
+  room Julietta
+  room LinkBedroom
+  room LinkEntry
+  room LinkSmithy
+  room Percy
+  room Romeo
+  room Stockwell
+  room WindTribe
```

Comment:
Is this Strato's house in town? Because HouseInteriors2.WindTribe suggests the tower (and is therefore very confusing).

Reply:
changed them!